### PR TITLE
Explaining how to disable all globs

### PR DESCRIPTION
**Pull request by adrianvmsft**
# Title

On the title describe
what you've fixed (or created) with this Pull Request (PR).

## Summary

Providing more information on how to disable the different types of implicit globs and explaining how to disable all globs.

Fixes #Issue_Number

>Note: The "Fixes #nnn" syntax in the PR description causes
>GitHub to automatically close the issue when this PR is merged.
> Remove that line if you don't have issues associated with this
> PR. Click on the Guidelines for Contributing link above for details.

## Details

Explain your changes, and why you made them. If that
information is already available in the issue referenced
above, just referencing the issue is preferred to copying
the text.

This may not be necessary depending on the scope of the PR 
changes. For example, "fix typo in introduction.md" is
sufficient to describe that PR.

## Suggested Reviewers

If you know who should review this, use '@' to request a review.

### CODE REVIEW

File: docs/core/tools/csproj.md [hunk 1]
```diff
@@ -66,6 +66,15 @@ Setting this property to `false` will override implicit inclusion and the behavi
 
 This change does not modify the main mechanics of other includes. However, if you wish to specify, for example, some files to get published with your app, you can still use the known mechanisms in *csproj* for that (for example, the `<Content>` element).
 
+`<EnableDefaultCompileItems>` only disables `Compile` globs but doesn't affect other globs, like the implicit `None` glob which also applies to \*.cs items. Because of that, `Solution Explorer` will continue show \*.cs items as part of the project, included as `None` items. In a similar way, you can use `<EnableDefaultNoneItems>` to disable the implicit `None` glob.
```
Comment (mairaw) [inline docs/core/tools/csproj.md]: missing comma before which

UI items such as Solution Explorer need to be formated in bold

File: docs/core/tools/csproj.md [hunk 2]
```diff
@@ -66,6 +66,15 @@ Setting this property to `false` will override implicit inclusion and the behavi
...
+
+If you would like to disable **all implicit globs**, you can set the `<EnableDefaultItems>` to `false`:
```
Comment (mairaw) [inline docs/core/tools/csproj.md]: If you would like to disable -> To disable

you can set the `<EnableDefaultItems>` to `false` -> set the `<EnableDefaultItems>` property to `false` as in the following example:

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @adrianvmsft. LGTM but left a few comments re. style guide rules.

File: docs/core/tools/csproj.md (hunk 1 above)
Comment (mairaw) [inline docs/core/tools/csproj.md]: also please change ms.date value on line 7 to today's date

**Comment by mairaw** [comment]
Thanks @adrianvmsft for making the fixes. @dsplaisted can you take a look at this PR?

/cc @KathleenDollard

**Review by livarcocc — APPROVED** [review]

**Review by mairaw — APPROVED** [review]

**Comment by mairaw** [comment]
Builds were stuck. Retrying again.

**Review by dsplaisted — APPROVED** [review]
This change is good.

Looking at the rest of the page, I think it could use some additional changes.  I don't understand what the recommendation for globbing is trying to say, and I don't think the distinction between exclude and remove expressions for the globs is helpful.

**Comment by mairaw** [comment]
I'll merge this and we can discuss improvements to the CLI docs when we meet @dsplaisted.

**Pull request merged**